using-valtimo/plugin/portaaltaak/configure-portaaltaak-plugin.md

```diff
@@ -0,0 +1,118 @@
+# Using the Portaal taak Plugin
+
+The Portaal Taak plugin is used to generate "taak" (task) objects within the Objecten API for a BPMN 
+user task. As a result, such tasks can be completed via external applications, such as the NL Portal. This feature 
+empowers Valtimo to assign tasks to other systems that are more adept at performing them. Furthermore, it enables users 
+who lack access to the Valtimo frontend to access these tasks.
+
+## How does the plugin work
+
+The lifecycle of a portaal taak is as follows:
+1. When a process instance reaches a user task configured with the [Create Portaal taak](#create-portaal-taak) action
+an instance of the task is created in Valtimo as usual. Additionally a taak object is created in the Objecten API of
+object type `Taak`. The status of the taak object will be `open`. The taak object includes information on who the 
+intended recipient is. This is the person or organization that is supposed to complete the task.
+2. External applications, such as the NL Portal, can retrieve the task information and offer a way to 
```

Comment:
NL Portal isn't something that has been explained yet. I think giving examples is good, I just don't know if it makes sense to mention the NL Portal, since the intention was to not document portal related stuff (obviously stuff that's in Valtimo is the exception to this).